### ISSUE TITLE
[CLI] Log grid image with wrong color

### ISSUE THREAD

**Issue by honguyenhaituan** [wandb/client#2309]
**Description**
I want to log a grid Image make by torchvision.utils.make_grid but the visualize image doesn't have the right color.

**Wandb Image**

**How to reproduce**
Fist I write log image like this:
```
if opt.save_compare_image:
    for _img, _att_img in zip(image, att_img):
        image_compare = make_grid([_img, _att_img - _img, _att_img])
        logger_compare_img.append(logger.wandb.Image(image_compare))
        save_image(image_compare, os.path.join(save_dir, "compare_img", "%i.png" % len(pred)))
```
and the resulting image 
![image](https://user-images.githubusercontent.com/85393497/122713950-b9d80e00-d290-11eb-9be6-922298f78af7.png)
Then so I change the way init Image by the path:
```
if opt.save_compare_image:
    for _img, _att_img in zip(image, att_img):
        image_compare = make_grid([_img, _att_img - _img, _att_img])
        save_image(image_compare, os.path.join(save_dir, "compare_img", "%i.png" % len(pred)))
        logger_compare_img.append(logger.wandb.Image(os.path.join(save_dir, "compare_img", "%i.png" % len(pred))))

```
and results
![image](https://user-images.githubusercontent.com/85393497/122714157-150a0080-d291-11eb-9f7a-be15525e756e.png)

**Environment**
- Environment: Google Colab

**Comment by vanpelt** [comment]
@honguyenhaituan this is an issue with your data / `make_grid` it looks like your data isn't normalized in the same way across the different images you're displaying the in the grid.  Either make them all between 0 and 1 or 0 and 255

**Comment by honguyenhaituan** [comment]
@vanpelt I see in the data_types.py have this line (1685 - 1687)
```
data = vis_util.make_grid(data, normalize=True)
self._image = pil_image.fromarray(
    data.mul(255).clamp(0, 255).byte().permute(1, 2, 0).cpu().numpy()
```

And this is a description of torch.utils.make_grid 
```
value_range (tuple, optional): tuple (min, max) where min and max are numbers,
then these numbers are used to normalize the image. By default, min and max
are computed from the tensor.
```
Then the problem occurs when my Image in the range [0, 1] but max and min does not equal to 1 and 0.

**Comment by vanpelt** [comment]
@honguyenhaituan this code path is only reached if you pass a torch tensor into `wandb.Image`, if you want more control over how the images are created just call `vis_util.make_grid` yourself and pass a PIL image into `wandb.Image`.  We're just trying to do the most sane thing when we see a pytorch tensor, and according to the [docs](https://pytorch.org/vision/stable/utils.html#torchvision.utils.make_grid).

> normalize (bool, optional) – If True, shift the image to the range (0, 1), by the min and max values specified by range. Default: False.
